src/constants.js

```diff
@@ -470,6 +470,16 @@ export const ACTIVITY_FILTER_OPTION_RESOLVED: 'resolved' = 'resolved';
 export const ACTIVITY_FILTER_OPTION_TASKS: 'tasks' = 'tasks';
 export const ACTIVITY_FILTER_OPTION_UNRESOLVED: 'open' = 'open';
 
+/* ------------------ File Activity Types ------------------ */
+export const FILE_ACTIVITY_TYPE_ANNOTATION: 'annotation' = 'annotation';
+export const FILE_ACTIVITY_TYPE_APP_ACTIVITY: 'app_activity' = 'app_activity';
+export const FILE_ACTIVITY_TYPE_COMMENT: 'comment' = 'comment';
+export const FILE_ACTIVITY_TYPE_TASK: 'task' = 'task';
+export const FILE_ACTIVITY_TYPE_VERSION: 'versions' = 'versions';
+
+/* ------------------ File Type ------------------ */
+export const FILE_TYPE_ACTIVITY: 'activity' = 'activity';
```

Comment:
Currently it can only be of type `activity` as far as I know. This is coming from the backend. Are you suggesting to name the typing something other than `FileActivityType`?

Reply:
it was specifically this name that confused me: `FILE_TYPE_ACTIVITY` under the `--- File Type ---` section

I would've assumed files types would be something like:
```
export const FILE_TYPE_IMAGE: 'image' = 'image';
export const FILE_TYPE_VIDEO: 'video' = 'video';
// or
export const FILE_TYPE_JPEG: 'jpeg' = 'jpeg';
```

but if this is how it aligns in the backend and others are familiar with it, we can leave this

Reply:
Yeah looked at it again, and its pretty confusing. I removed the naming since it's only 'activity' for now. We can define a type in a different PR which uses it